build.sbt

```diff
@@ -9,18 +9,17 @@ addCommandAlias("ci-docs", "github; mdoc; headerCreateAll")
 addCommandAlias("ci-publish", "github; ci-release")
 
 ThisBuild / resolvers += Resolver.sonatypeRepo("snapshots")
-lazy val muV = "0.28.0+28-c32576e2-SNAPSHOT" // TODO update when mu-scala is released
+lazy val muV = "0.28.0+34-effa638e-SNAPSHOT" // TODO update when mu-scala is released
```

Comment:
```suggestion
```

Reply:
We still need this line because it's used to set the version of the Mu dependency in the scripted tests.